src/lib/serviceRoutinesV2/patchEntity.cpp

```diff
@@ -25,6 +25,9 @@
 #include <string>
 #include <vector>
 
+#include "logMsg/logMsg.h"
```

Comment:
Really needed in this file? I don't see any change in this file using the things added by these includes...

Reply:
I probably had an LM_T and then removed it.
These things happen during PRs.
The inclusion of logMsg.h doesn't hurt (except for a millisecond extra during compilation)

Reply:
I agree it doesn't hurt, but let's try to keep files tidy. If an include is doesn't use by a file, the it should be removed (if I'm remembering correctly, I think we have some rules about that in style guide).